BrainPortal/app/views/portal/welcome.html.erb

```diff
@@ -167,14 +167,14 @@
       <strong>Your current time:</strong> <%= to_localtime(Time.now, :datetime) %><BR>
     </p>
     <p>
-       <strong>Projects you belong to:</strong>  <%= array_to_table(@groups, :table_class => 'simple', :cols => 4) { |g,r,c| link_to_group_if_accessible(g) } %>
+      <strong>Tools available to you (out of <%= link_to @tool_names.count , available_path %>):</strong><br/>
```

Comment:
WHy do we have the same link twice? (line 180 and 174). The one at line 170 is not necessary, it could be be the number.

Reply:
That was the point of the PR - To increase visibility of the full tool list. For an average user the link to the full list on line 180 is hard to find, not everyone has a habbit of scrolling welcome page. The one on 170 is much higher and a greater chance to be discovered by users. 

See the issue #1409 for a snapshot and discussion. 

In my opinion repeating important info several time is a good UX practice which we can use more often. But if you uncomfortable with it, maybe it is better just raise line 180 above 174, perhaps with a minor verbiage change?